.github/workflows/check-for-changes.yml

```diff
@@ -22,5 +24,17 @@ jobs:
       - name: show files
         run: cat updatedfiles.txt
       - name: Have files changed
-        if: steps.git-check.outputs.changes_present == 'false'
-        run: exit 1
\ No newline at end of file
+        if: steps.checkForChanges.outputs.changes_present == 'false'
+        run: echo "No changes needed to be synchronised"
+  sync:
+    needs: checkForChanges
```

Comment:
what's the actual standard for naming targets? camel casing just looks a bit surprising in the logs

Reply:
No standard as far as I'm aware, examples appear mixed. Would you prefer we used hyphens or underscores instead?

Reply:
`-`